runtime/kusama/src/lib.rs

```diff
@@ -633,7 +633,7 @@ parameter_types! {
 impl pallet_treasury::Config for Runtime {
 	type PalletId = TreasuryPalletId;
 	type Currency = Balances;
-	type ApproveOrigin = EnsureRoot<AccountId>;
+	type ApproveOrigin = EitherOf<EnsureRoot<AccountId>, Treasurer>;
 	type RejectOrigin = EnsureRoot<AccountId>;
```

Comment:
I think Treasurer should also be able to reject. Let me know if I should update this as well.